multidict/_multidict.pyx

```diff
@@ -224,37 +224,34 @@ cdef class _Base:
 
 
 cdef class MultiDictProxy(_Base):
+    _proxy_classes = (MultiDict, MultiDictProxy)
+    _base_class = MultiDict
 
     def __init__(self, arg):
         cdef _Base base
-        if not isinstance(arg, (MultiDict, MultiDictProxy)):
+        if not isinstance(arg, self._proxy_classes):
             raise TypeError(
-                'ctor requires MultiDict or MultiDictProxy instance'
+                'ctor requires {} instance'
                 ', not {}'.format(
+                    ' or '.join(self._proxy_classes),
                     type(arg)))
 
         base = arg
         self._items = base._items
 
+    def __reduce__(self):
+        raise pickle.PicklingError("Unsupported")
```

Comment:
look at my example about, it actually raises: `_pickle.PicklingError: Can't pickle <class 'mappingproxy'>: attribute lookup mappingproxy on builtins failed`